### PR TITLE
Introduce Trust Service Criteria, based on the Trust Service Principl…

### PR DESCRIPTION
**Pull request by jmccarthy**
…es. Used in the assessment of SOC 2 compliance.

### CODE REVIEW

File: tsc-2017.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
+name: TSC
+CC1.1:
```
Comment (shawndwells) [inline tsc-2017.yaml:2]: Should these names have spaces, e.g. ``CC 1.1``?

Attempted to check, but looks like the TSP's are behind a paywall.

File: tsc-2017.yaml [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+  family: CC1
+  name: Integrity and Ethics
+  description: The entity demonstrates a commitment to integrity and ethical values
+CC1.2:
+  family: CC1
+  name: Board Independence
+  description: The board of directors demonstrates independence from management and exercises oversight of the development and performance of internal control
```
Comment (shawndwells) [inline tsc-2017.yaml]: End of line periods are needed.

**Review by shawndwells — CHANGES_REQUESTED** [review]
Thanks @jmccarthy ! 

Because TSP appears to be behind a paywall, wasn't able to find if the control should have a space in the name (CC1.1 vs CC 1 .1). Can you clarify?

Small nitpick that the ``description`` element should be authored like a sentence. Need end of line periods.

File: tsc-2017.yaml (hunk 2 above)
Comment (jmccarthy) [reply tsc-2017.yaml]: Will add

File: tsc-2017.yaml (hunk 1 above)
Comment (jmccarthy) [reply tsc-2017.yaml:2]: Just checked: they seem to be pervasively referenced without the space (definitely CC1.5, PI1.6, etc) in both the source material as well as the auditor documents I have reviewed. Will stick with this format.

**Review by shawndwells — APPROVED** [review]

**Comment by shawndwells** [comment]
Thanks, @jmccarthy !

**Pull request merged**